web/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "ss-web",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "clean": "",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "@bcgov/bootstrap-theme": "https://github.com/bcgov/bootstrap-theme/releases/download/v1.1.3/bcgov-bootstrap-theme-1.1.3.tgz",
+    "@types/underscore": "^1.10.0",
+    "axios": "^0.19.2",
+    "bootstrap": "^4.4.1",
+    "bootstrap-vue": "^2.12.0",
+    "core-js": "^3.6.5",
+    "intersection-observer": "^0.10.0",
+    "jquery": "^3.5.1",
+    "regenerator-runtime": "^0.13.5",
+    "underscore": "^1.10.2",
+    "vue": "2.6.11",
```

Comment:
This was giving me grief:
```
Module Error (from ./node_modules/vue-loader/lib/index.js):


Vue packages version mismatch:

- vue@2.6.11 (/opt/app-root/src/node_modules/vue/dist/vue.runtime.common.js)
- vue-template-compiler@2.6.12 (/opt/app-root/src/node_modules/vue-template-compiler/package.json)

This may cause things to work incorrectly. Make sure to use the same version for both.


```

Looks like we can upgrade to vue 2.6.12 safely, but problems like this are why I don't like to use fuzzy versions in package.json files.

Reply:
Same problem. Vue 3 was released today, I'd like to upgrade to that.

Reply:
@seeker25 Cool, good call.